### PR TITLE
Changes for moving the oam-hello-helidon example and test to verrazzano

### PR DESCRIPTION
**Pull request by lake-of-dreams**
Changes for moving the oam-hello-helidon example and test to verrazzano

### CODE REVIEW

**Review by markxnelson — APPROVED** [review]

File: examples/hello-helidon/README.md [hunk 1]
```diff
@@ -1,153 +1,128 @@
...
+   * **Local Testing with a Browser** 
+   
+     Temporarily modify the `/etc/hosts` file (on Mac or Linux)
+     or `c:\Windows\System32\Drivers\etc\hosts` file (on Windows 10), 
+     to add an entry mapping `hello-helidon.example.com` to the ingress gateway's `EXTERNAL-IP` address.
+     For example:
+     ```
+     11.22.33.44 hello-helidon.example.com
+     ```
+     Then, you can access the application in a browser at `http://hello-helidon.example.com/greet`
```
Comment (rosemarymarano) [inline examples/hello-helidon/README.md:70]: please put a period at the end of this sentence.

File: examples/hello-helidon/README.md [hunk 2]
```diff
@@ -1,153 +1,128 @@
...
+
+   * **Using your own DNS Name:**
```
Comment (rosemarymarano) [inline examples/hello-helidon/README.md:72]: DNS Name -> DNS name

File: examples/hello-helidon/README.md [hunk 3]
```diff
@@ -1,153 +1,128 @@
...
+
```
Comment (rosemarymarano) [inline examples/hello-helidon/README.md:73]: Please put a period at the end of each of the following bulleted items.

File: examples/hello-helidon/README.md [hunk 4]
```diff
@@ -1,153 +1,128 @@
...
+     11.22.33.44 hello-helidon.example.com
+     ```
+     Then, you can access the application in a browser at `http://hello-helidon.example.com/greet`
+
+   * **Using your own DNS Name:**
+   
+     * Point your own DNS name to the ingress gateway's `EXTERNAL-IP` address
+     * In this case, you would need to edit the `hello-helidon-app.yaml` file 
+       to use the appropriate value under the `hosts` section (such as `yourhost.your.domain`), 
+       before deploying the hello-helidon application.
```
Comment (rosemarymarano) [inline examples/hello-helidon/README.md:77]: deploying the hello-helidon application -> deploying the `hello-helidon` application

File: examples/hello-helidon/README.md [hunk 5]
```diff
@@ -1,153 +1,128 @@
...
+     * Then, you can use a browser to access the application at `http://<yourhost.your.domain>/greet`
+   
+3. ## Troubleshooting
+    
+4. Verify that the application configuration, domain and ingress trait all exist.
```
Comment (rosemarymarano) [inline examples/hello-helidon/README.md:82]: domain -> domain, (serial comma)

File: examples/hello-helidon/README.md [hunk 6]
```diff
@@ -1,153 +1,128 @@
...
+   ```
+   kubectl get ApplicationConfiguration -n oam-hello-helidon
+   kubectl get Domain -n oam-hello-helidon
+   kubectl get IngressTrait -n oam-hello-helidon
+   ```   
+
+5. Verify that the hello-helidon service pods are successfully created and transition to the ready state.
```
Comment (rosemarymarano) [inline examples/hello-helidon/README.md:89]: hello-helidon -> `hello-helidon` (code font)

File: examples/hello-helidon/README.md [hunk 7]
```diff
@@ -1,153 +1,128 @@
...
+   Note that this may take a few minutes and that you may see some of the services terminate and restart.
+   ```
+    kubectl get pods -n oam-hello-helidon
+   
+    NAME                                      READY   STATUS    RESTARTS   AGE
+    hello-helidon-workload-676d97c7d4-wkrj2   2/2     Running   0          5m39s
+   ``` 
+6. A variety of endpoints are available to further explore the logs, metrics, etc assoicated with
```
Comment (rosemarymarano) [inline examples/hello-helidon/README.md:97]: etc  -> and such,

File: examples/hello-helidon/README.md [hunk 8]
```diff
@@ -1,153 +1,128 @@
...
+5. Verify that the hello-helidon service pods are successfully created and transition to the ready state.
+   Note that this may take a few minutes and that you may see some of the services terminate and restart.
+   ```
+    kubectl get pods -n oam-hello-helidon
+   
+    NAME                                      READY   STATUS    RESTARTS   AGE
+    hello-helidon-workload-676d97c7d4-wkrj2   2/2     Running   0          5m39s
+   ``` 
+6. A variety of endpoints are available to further explore the logs, metrics, etc assoicated with 
+the deployed hello-helidon application.  Accessing them may require the following:
```
Comment (rosemarymarano) [inline examples/hello-helidon/README.md:98]: the deployed hello-helidon application -> the deployed Hello World Helidon application

File: examples/hello-helidon/README.md [hunk 9]
```diff
@@ -1,153 +1,128 @@
...
+
+    - The telemetry password: Run this command to get the password that was generated for the telemetry components:
```
Comment (rosemarymarano) [inline examples/hello-helidon/README.md:100]: The telemetry password: -> The telemetry password. (period, or eliminate this sentence as it is repeated in the next sentence)

File: examples/hello-helidon/README.md [hunk 10]
```diff
@@ -1,153 +1,128 @@
...
+        ```
+        kubectl get secret --namespace verrazzano-system verrazzano -o jsonpath={.data.password} | base64 --decode; echo
+        ``` 
+        The associated username is 'verrazzano'
```
Comment (rosemarymarano) [inline examples/hello-helidon/README.md:104]: username -> user name 
'verrazzano' ->`verrazzano` (code font)

File: examples/hello-helidon/README.md [hunk 11]
```diff
@@ -1,153 +1,128 @@
...
+   
+    - You will have to accept the certificates associated with the endpoints
```
Comment (rosemarymarano) [inline examples/hello-helidon/README.md:106]: endpoints -> endpoints. (period)

File: examples/hello-helidon/README.md [hunk 12]
```diff
@@ -1,153 +1,128 @@
...
+    verrazzano-console-ingress   <none>   verrazzano.default.140.238.94.217.xip.io                 140.238.94.217   80, 443   7d2h
+    vmi-system-api               <none>   api.vmi.system.default.140.238.94.217.xip.io             140.238.94.217   80, 443   7d2h
+    vmi-system-es-ingest         <none>   elasticsearch.vmi.system.default.140.238.94.217.xip.io   140.238.94.217   80, 443   7d2h
+    vmi-system-grafana           <none>   grafana.vmi.system.default.140.238.94.217.xip.io         140.238.94.217   80, 443   7d2h
+    vmi-system-kibana            <none>   kibana.vmi.system.default.140.238.94.217.xip.io          140.238.94.217   80, 443   7d2h
+    vmi-system-prometheus        <none>   prometheus.vmi.system.default.140.238.94.217.xip.io      140.238.94.217   80, 443   7d2h
+    vmi-system-prometheus-gw     <none>   prometheus-gw.vmi.system.default.140.238.94.217.xip.io   140.238.94.217   80, 443   7d2h
+    ```  
+    
+    Some of the endpoints available (leveraging the ingress host information above):
```
Comment (rosemarymarano) [inline examples/hello-helidon/README.md:122]: leveraging  -> using

File: examples/hello-helidon/README.md [hunk 13]
```diff
@@ -1,153 +1,128 @@
...
+    
+    | Description| Address | Credentials |
+    | --- | --- | --- |
```
Comment (rosemarymarano) [inline examples/hello-helidon/README.md:125]: Please put all the following URLs, such as `https://[vmi-system-kibana ingress host]`, in code font

**Review by rosemarymarano — COMMENTED** [review]
Some edits and suggestions for you to address.

**Pull request merged**